src/pages/[platform]/build-a-backend/storage/upload/index.mdx

```diff
@@ -106,38 +90,52 @@ try {
     key: filename,
     data: file,
     options: {
-      accessLevel: 'protected'
+      accessLevel: 'guest', // defaults to `guest` but can be 'private' | 'protected' | 'guest'
+      onProgress // Optional progress callback.
```

Comment:
Should we specify an example, or can assume this symbol exists already?

```suggestion
      onProgress // Optional progress callback.
```

Reply:
these snippet exactly as what it is right now. we might wanna change that everywhere